modules/Vtiger/ExecuteFunctions.php

```diff
@@ -693,6 +693,12 @@
 			);
 		}
 		break;
+	case 'getModuleFieldsInfo':
```

Comment:
I don't want to expose this functionality to JS. They can use web service to get that information. Leave just the change above please